test/integration/full/contrast/blending.js

```diff
@@ -1,18 +1,173 @@
+var isIE11 = axe.testUtils.isIE11;
+
 describe('color-contrast blending test', function() {
   var include = [];
   var resultElms = [];
   var expected = [
+    // normal
     'rgb(223, 112, 96)',
     'rgb(255, 128, 128)',
     'rgb(191, 223, 191)',
     'rgb(125, 38, 54)',
     'rgb(179, 38, 0)',
     'rgb(179, 0, 77)',
-    'rgb(143, 192, 80)',
```

Comment:
why this change? Seems like a rounding difference of some sort.

Reply:
I believe this may of impacted by https://github.com/dequelabs/axe-core/pull/3226#discussion_r752386049 and the threshold of rounding was impacted